src/aeacus.go

```diff
@@ -155,12 +159,29 @@ func main() {
 				Aliases: []string{"r"},
 				Usage:   "Prepare the image for release",
 				Action: func(c *cli.Context) error {
-					runningPermsCheck()
 					parseFlags(c)
+					runningPermsCheck()
+					if !yesEnabled {
+						confirmPrint("Are you sure you want to being the image releasing process?")
```

Comment:
uhh english

Reply:
i think that was a typo for `begin`

Reply:
Oops yeah it be like that sometimes, good catch